app/models/zone.rb

```diff
@@ -111,24 +114,10 @@ def log_collection_active_recently?(since = nil)
     miq_servers.any? { |s| s.log_collection_active_recently?(since) }
   end
 
-  def host_ids
```

Comment:
Do we lose this method?  Does rails translate the has_many :hosts to a `host_ids` method via inflection?